content/zh/docs/reference/config-api/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Configuration APIs 
```

Comment:
```suggestion
title: 配置 API
```

Reply:
Thanks for your advice. I have modified it